.claude/PRPs/plans/completed/persona-foundry-phase-1-engine-core.plan.md

```
@@ -0,0 +1,1067 @@
+  Validation Commands below) — this is the task where the whole engine gets
+  proven, not a unit test.
+
+### Task 9: Wire up `package.json` and `.gitignore`
+- **ACTION**: Add the npm script and gitignore entry.
+- **IMPLEMENT**:
+  ```json
+  "persona": "tsx scripts/seed-persona.ts"
+  ```
+  placed near `"db:seed-templates"` in `package.json`'s `scripts` block.
+  In `.gitignore`, under the existing `# playwright` section, add:
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)